lib/plugins/audit.js

```diff
@@ -50,6 +51,7 @@ function getResponseHeaders(res) {
  * the audit log object programmatically
  * @param {boolean} [opts.printLog=true] - Whether to print the log
  * via the logger.
+ * @param {Array<String>} [opts.customFields=[]] - Custom fields for logger
  * @returns {Function} Handler
  * @fires audit when an audit log has been generated
  * @example
```

Comment:
could you please provide a JSDoc example?

Reply:
Should i change `Array<String>` to `string[]`?

Reply:
Yes, please. Also you can add a new example to JSDoc like:
```
* @example
* <caption>Some text</caption>
* Some JS Code